demisto_sdk/commands/common/git_util.py

```diff
@@ -422,21 +422,37 @@ def check_if_remote_exists(self, remote):
 
         return remote in self.repo.remotes
 
-    def _handle_prev_ver(self, prev_ver):
+    def handle_prev_ver(self, prev_ver: str = ''):
         # check for sha1 in regex
         sha1_pattern = re.compile(r'\b[0-9a-f]{40}\b', flags=re.IGNORECASE)
-        if sha1_pattern.match(prev_ver):
+        if prev_ver and sha1_pattern.match(prev_ver):
             return None, prev_ver
 
-        if '/' in prev_ver:
-            remote = prev_ver.split('/')[0]
+        if prev_ver and '/' in prev_ver:
+            remote, branch = prev_ver.split('/', 1)
             remote = remote if self.check_if_remote_exists(remote) else str(self.repo.remote())
-            branch = prev_ver.split('/')[1]
 
         else:
             remote = str(self.repo.remote())
-            branch = prev_ver
-
+            branch = ''
+            if prev_ver:
+                branch = prev_ver
+            else:
+                # Try to get the main branch
+                for current_remote in self.repo.remotes:
+                    for current_remote_ref in current_remote.refs:
+                        current_remote_ref_str = str(current_remote_ref)
+                        print('*** , ' + str(type(current_remote_ref)) + ', current_remote_ref: ' + str(current_remote_ref))
```

Comment:
are these prints relevant in production?

Reply:
Removed